src/routes/+page.svelte

```diff
@@ -313,7 +313,7 @@
     class="banner flex flex-col gap-y-16 justify-center items-center px-6 lg:px-20 py-16"
 >
     <div
-        class="container min-h-[633px] flex gap-24 flex-col lg:flex-row justify-start items-center"
+        class="container min-h-[633px] flex gap-24 flex-col lg:flex-row lg:gap-48 justify-start items-center"
```

Comment:
`gap-48` translates to gap of 192px, which is a way bigger gap than the 111px gap specified in Figma. This should be `gap-28` which is 112px


And I'm not sure what happened. but the image is too close to the left margin of the page now. We should align the whole content to the center of the banner
<img width="1512" alt="image" src="https://github.com/user-attachments/assets/0dfc9cac-27ab-4a0e-b9c4-113818da6fca">